content/coding_standards/ada_coding_standards/chapters/guidelines/concurrency/con03_avoid_shared_variables_for_inter-task_communication.rst

```diff
@@ -0,0 +1,76 @@
+-------------------------------------------------------------
+Avoid Shared Variables for Inter-task Communication (CON03)
+-------------------------------------------------------------
+
+.. include:: ../../../../global.txt
+
+**Level** :math:`\rightarrow` Advisory
+
+Category
+   :Safety: :math:`\checkmark`
+   :Cyber: :math:`\checkmark`
+
+Goal
+   :Maintainability: :math:`\checkmark`
+   :Reliability: :math:`\checkmark`
+   :Portability: :math:`\checkmark`
+   :Performance: :math:`\checkmark`
+   :Security:
+
+**Remediation** :math:`\rightarrow` High
+
+**Verification Method** :math:`\rightarrow` GNATcheck rule:
+:rule:`Volatile_Objects_Without_Address_Clauses`
+
++++++++++++
+Reference
++++++++++++
+
+`Ada RM D.13 - The Ravenscar Profile
+<http://ada-auth.org/standards/12rm/html/RM-D-13.html>`_
+
++++++++++++++
+Description
++++++++++++++
+
+Although the Ravenscar and Jorvik profiles allow the use of shared variables
+for inter-task communication, such use is less robust and less reliable than
+encapsulating shared variables within protected objects.
+
+++++++++++++++++++++++++++++++++++++++++++++++++
+Applicable vulnerability within ISO TR 24772-2
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+* 6.56 Undefined behaviour [EWF]
+
++++++++++++++++++++++++++++
+Noncompliant Code Example
++++++++++++++++++++++++++++
+
+.. literalinclude:: examples/con03.ads
+  :language: Ada
+  :lines: 5-7
+  :dedent: 3
+
+Note that variables marked as :ada:`Atomic` are also :ada:`Volatile`, per the
+`Ada RM C.6/8(3) - Shared Variable Control
+<http://www.ada-auth.org/standards/12rm/html/RM-C-6.html>`_
```

Comment:
Hi @frank-at-adacore 

Just in case you did not see it, the CI triggered an error on this line:

con03_avoid_shared_variables_for_inter-task_communication.rst:55:hardcoded link 'http://www.ada-auth.org/standards/12rm/html/RM-C-6.html' could be replaced by an extlink (try using ':arm:`Ada RM C.6\\/8\\(3\\) \\- Shared Variable Control <C-6>`' instead)

Reply:
Fixed